src/service/Domain/FeatureFlag.cs

```diff
@@ -13,6 +13,8 @@ public class FeatureFlag
         public string Description { get; set; }
         [JsonProperty("enabled")]
         public bool Enabled { get; set; }
+        [JsonProperty("enableIncrementalRings")]
+        public bool EnableIncrementalRings { get; set; }
```

Comment:
To maintain consistency rename to `IncrementalRingsEnabled`

Reply:
done